artemis-server/src/main/java/org/apache/activemq/artemis/spi/core/security/ActiveMQJAASSecurityManager.java

```diff
@@ -80,31 +81,41 @@ public ActiveMQJAASSecurityManager(String configurationName, String certificateC
    }
 
    @Override
-   public boolean validateUser(String user, String password) {
```

Comment:
I just checked and Wildfly won't build after this change.

I know at least one user who was writing his own SecurityManager, so this would break not just wildfly.

it's part of the SPI, so we can't change it without breaking API. 

as we talked, make a SecurityManager3 and add what you need there, check of instanceof of the new corrent interface.